eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/DateAndTimeConverter.php

```diff
@@ -123,7 +123,7 @@ public function toFieldDefinition(StorageFieldDefinition $storageDef, FieldDefin
                 $data = array(
                     'rfc850' => null,
                     'timestamp' => $date->getTimestamp(), // @deprecated timestamp is no longer used and will be removed in a future version.
-                    'timestring' => sprintf('%+d seconds', $date->getTimestamp() - time()),
+                    'timestring' => $dateInterval->format('%y years, %m months, %d days, %h hours, %i minutes, %s seconds'),
```

Comment:
No, they are the same in that regard. Both will cache a descriptive/relative timestring ("2 days, 3 hours"), rather than a timestamp. Only when you instantiate the field will this be evaluated/translated into an actual DateTime.